ts-eks/test/pulumi/index.ts

```diff
@@ -15,14 +15,32 @@ const eksPolicy: StackValidationPolicy = {
         const cluster = eksClusters[0].asType(aws.eks.Cluster)
 
         const nodeGroups = args.resources.filter(r => r.isType(aws.eks.NodeGroup));
-        if (nodeGroups.length !== 1) {
-            reportViolation(`Node groups: expected 1 but found ${nodeGroups.length}`);
+        if (nodeGroups.length !== 2) {
```

Comment:
this supposed to be 3?

Reply:
nope just 2, we create the minimum number of autoscaling groups to satisfy every exec units needs.